minecraft/java/purpur/README.md

```diff
@@ -0,0 +1,77 @@
+See https://github.com/pl3xgaming/purpur for additional information. I copy-pasted this readme from there.
```

Comment:
I don't need their entire README in my repo. Just a link to theirs is fine.

Reply:
Done!